pkg/engine/masterarmresources.go

```diff
@@ -192,9 +193,11 @@ func createKubernetesMasterResourcesVMSS(cs *api.ContainerService) []interface{}
 	}
 
 	if isKMSEnabled {
+		// TODO (aramase) remove storage account creation as part of kms plugin v0.0.11
```

Comment:
should we do this in this PR?

Reply:
The current release `v0.0.10` is still dependent on the storage account to exist. I'll remove it when we cut a kms release `v0.0.11` and update the version in aks-engine.